EpsilonCCS/Inc/Data/DriverControlData.h

```diff
@@ -15,11 +15,10 @@
 #define INTERIOR_MASK 0x40
 
 // Music Inputs
-#define AUX_MASK 0x01
-#define VOLUME_UP_MASK 0x02
-#define VOLUME_DOWN_MASK 0x04
-#define NEXT_SONG_MASK 0x08
-#define PREV_SONG_MASK 0x10
+#define VOLUME_UP_MASK 0x01
+#define VOLUME_DOWN_MASK 0x02
+#define NEXT_SONG_MASK 0x04
+#define PREV_SONG_MASK 0x8
```

Comment:
Use `0x08` for consistency with the rest.

Reply:
Oh, oops